vignettes/FDZ_Data_Cleaning.Rmd

```diff
@@ -112,7 +94,7 @@ dat4 <- multiChar2fac(dat3, vars = c("ID_name", "info", "subjfav", "birth"), var
 ```
 
 # -----------------------------------------------------------------------------
-## Empty variables already in IZA (with loop)
+## Empty variables already in IZA
 If variables are of strong data protection concern, they are already emptied directly in the IZA. This is achieved with the `emptyTheseVariables()` function.
 ```{r emptyTheseVariables}
 dat3_empty <- c("info",
```

Comment:
Maybe add clarification, why we go back to dat3 instead of dat4.

Reply:
I set it back to dat4, there was no particular reason for going back